.vscode/extensions.json

```diff
@@ -1,3 +1,8 @@
 {
-  "recommendations": ["esbenp.prettier-vscode", "dbaeumer.vscode-eslint"]
+  "recommendations": [
+    "esbenp.prettier-vscode",
+    "dbaeumer.vscode-eslint",
+    "streetsidesoftware.code-spell-checker",
+    "redhat.vscode-yaml"
```

Comment:
@backlineint any opposition to recommending these plugins? I find them very helpful, mostly the spellchecker.